app/mailers/rent_created_mailer.rb

```diff
@@ -3,6 +3,15 @@ class RentCreatedMailer < ApplicationMailer
 
   def created(rent_id)
     @current_rent = Rent.find(rent_id)
-    mail(to: @current_rent.user.email, subject: 'Created rent data')
+
+    I18n.with_locale(set_locale) do
+      mail(to: @current_rent.user.email, subject: I18n.t(:subject_email))
+    end
+  end
+
+  private
+
+  def set_locale
+    @current_rent.user.try(:locale) || I18n.locale_default
```

Comment:
Since ruby 2.3 you should use `@current_rent.user&.locale` instead of `Object#try`

Reply:
done!